primitives/rpc/debug/src/single.rs

```diff
@@ -31,7 +31,11 @@ use sp_std::{collections::btree_map::BTreeMap, vec::Vec};
 #[derive(Clone, Copy, Eq, PartialEq, Debug, Encode, Decode)]
 pub enum TraceType {
 	/// Classic geth with no javascript based tracing.
-	Raw,
+	Raw {
+		disable_storage: Option<bool>,
```

Comment:
It may be simpler to have a `bool` with `#[serde(default)]` attribute.

Reply:
Do you mean in `TraceParams`?